addons/im_livechat/static/tests/embed/chat_window.test.js

```diff
@@ -18,53 +16,18 @@ import {
 
 import { describe, test } from "@odoo/hoot";
 
+import { asyncStep, serverState, waitForSteps } from "@web/../tests/web_test_helpers";
 import { deserializeDateTime } from "@web/core/l10n/dates";
 import { getOrigin } from "@web/core/utils/urls";
-import {
-    asyncStep,
-    mountWithCleanup,
-    patchWithCleanup,
-    serverState,
-    waitForSteps,
-} from "@web/../tests/web_test_helpers";
 
 describe.current.tags("desktop");
 defineLivechatModels();
 
-test("save fold state of temporary live chats", async () => {
```

Comment:
Note: irrelevant as temporary threads are just discarded on reload after this PR.